src/api/json/catalog.json

```diff
@@ -7308,6 +7308,12 @@
       "fileMatch": ["**/*winutil*/config/preset.json"],
       "url": "https://json.schemastore.org/winutil-preset.json"
     },
+    {
+      "name": "GitHub Actions typing",
+      "description": "Describe types of inputs and outputs for GitHub Actions' actions",
+      "fileMatch": ["action-types.yml", "action-types.yaml"],
+      "url": "https://raw.githubusercontent.com/typesafegithub/github-actions-typing/refs/heads/main/github-actions-typing.schema.json"
```

Comment:
Do you really want this to point at `main` so that unreleased versions might be used, or maybe maintain some `latest` branch pointing to the latest released version instead?

Reply:
Good idea, let me create a `schema-latest` branch that will probably track the latest release of the action itself, but still we'll be able to set it to an arbitrary git ref to deploy fixes independently.

Reply:
```suggestion
      "url": "https://raw.githubusercontent.com/typesafegithub/github-actions-typing/refs/heads/schema-latest/github-actions-typing.schema.json"
```